sdk/aqueduct/integrations/mongodb_integration.py

```diff
@@ -0,0 +1,171 @@
+import json
+from typing import Any, Dict, List, Optional
+
+from aqueduct.artifacts import utils as artifact_utils
+from aqueduct.artifacts.base_artifact import BaseArtifact
+from aqueduct.artifacts.metadata import ArtifactMetadata
+from aqueduct.artifacts.table_artifact import TableArtifact
+from aqueduct.dag import DAG
+from aqueduct.dag_deltas import AddOrReplaceOperatorDelta, apply_deltas_to_dag
+from aqueduct.enums import ArtifactType, ExecutionMode, LoadUpdateMode
+from aqueduct.error import InvalidUserArgumentException
+from aqueduct.integrations.integration import Integration, IntegrationInfo
+from aqueduct.integrations.sql_integration import find_parameter_artifacts, find_parameter_names
+from aqueduct.operators import (
+    ExtractSpec,
+    MongoExtractParams,
+    Operator,
+    OperatorSpec,
+    RelationalDBLoadParams,
+    SaveConfig,
+)
+from aqueduct.utils import artifact_name_from_op_name, generate_uuid
+
+from aqueduct import globals
+
+
+class MongoDbCollectionIntegration(Integration):
+    _collection_name: str
+    _dag: DAG
+
+    def __init__(self, dag: DAG, metadata: IntegrationInfo, collection_name: str) -> None:
+        self._metadata = metadata
+        self._dag = dag
+        self._collection_name = collection_name
+
+    def find(
+        self,
+        *kargs: List[Any],
```

Comment:
nit: `kargs` seems like a misnomer, shouldn't it be just `args`? It's not taking in keyword arguments. Also I think you just need to type it as `Any`, since that type will apply to every elem in the list.